pkg/virtual/apiexport/controllers/apireconciler/apiexport_apireconciler_controller.go

```diff
@@ -76,7 +77,8 @@ func NewAPIReconciler(
 	}
 
 	if err := apiExportInformer.Informer().AddIndexers(cache.Indexers{
-		byWorkspace: indexByWorkspace,
+		byWorkspace:                  indexByWorkspace,
+		indexers.APIExportByIdentity: indexers.IndexAPIExportByIdentity,
```

Comment:
is this potentially already added?

Reply:
that was the bug - refresh & see the latest 😄